server/stt/io/feeding_services/stt_tt_content_api.py

```diff
@@ -139,7 +165,48 @@ def _fetch_tt_data(self, provider) -> List[Dict]:
         page_size = int(config.get("page_size", 50))
         max_pages = int(config.get("max_pages", 200))
         # safety cap to avoid runaway loops
-        timeout = int(config.get("timeout", 300))
+        timeout = int(config.get("timeout", 60))
+
+        def _coerce_bool(v):
+            if isinstance(v, bool):
+                return v
+            if isinstance(v, (int, float)):
+                return v != 0
+            if isinstance(v, str):
+                s = v.strip().lower()
+                if s in {"1", "true", "yes", "y", "on"}:
+                    return True
+                if s in {"0", "false", "no", "n", "off", ""}:
+                    return False
+            return False
+
+        # Optional incremental sync using 'trs' (timestamp since last run)
+        use_trs = _coerce_bool(config.get("use_trs", False))
```

Comment:
I wonder why making it optional, is it required not to be optional at some point?